styles.css

```diff
@@ -1,8 +1,57 @@
-/* font-family: 'Quicksand', sans-serif;
-COLORS
-light-yellow - #f7e4bf;
-light-blue - #78a7c6;
-dark-blue - #134d71;
-white - #ffffff;
-black - #000000;
-*/
\ No newline at end of file
+html {
```

Comment:
Something to think about long term: if multiple elements will have "display: flex" on them, consider having =>
.element-1, .element-2, .element-5, .element8 {
   display:flex;
}

and then => 

.element-1 {
  // random things specific to only this element
}
.element-2 {
  // random things specific to only this element
}